draft-ietf-httpbis-http2bis.xml

```diff
@@ -3474,6 +3474,13 @@
           an attacker if they are translated verbatim.  Any request or response that contains a
           character not permitted in a field value MUST be treated as <xref target="malformed">malformed</xref>.  Valid characters are defined by the <tt>field-content</tt> ABNF rule in <xref target="HTTP" section="5.5"/>.
         </t>
+        <t>
+          An intermediary that receives any field that requires removal before forwarding
+          (see <xref target="HTTP" section="7.6.1" />) must remove or replace those header fields when
+          forwarding messages, including between multiple messaging formats. Additionally, intermediaries
+          should take care when forwarding messages containing Content-Length fields to ensure that the
```

Comment:
```suggestion
          MUST validate any Content-Length fields before forwarding to ensure that the
```

Reply:
Hmm, I don't think that wording is acceptable: it implies that intermediaries are obligated to buffer all messages before they forward them on, which I don't think they'll do. We could say MUST validate any Content-Length fields _when_ forwarding, I suppose.